ba/js/lesson3/script.js

```diff
@@ -0,0 +1,196 @@
+//Task 1
+
+let start = prompt(" Enter start number");
+let end = prompt("Enter end number");
+let sum = 0;
+for (let i = +start; i < +end + 1; i++) {
```

Comment:
it better not to write it like that let start = +prompt(" Enter start number") or let start = Number(prompt(" Enter start number"));